### PR TITLE
Removed duplicate title tags and descriptions 

### LINKED ISSUE #4041: [SEO] Update Duplicate Title Tags and Meta descriptions
### Current Behavior
The pages listed below are using repeated `<title>` tags which confuse users as to which webpage they should follow and affects the search engines to determine which of a website's pages is relevant for a specific search query and which one should be prioritized in search results.

Duplicate meta descriptions make it difficult for search engines and users to differentiate between different webpages. It is better to have no meta description at all than to have a duplicate one.

Pages affected with duplicate `<title>` tags and meta descriptions -

1. https://layer5.io/cloud-native-management/meshmap/collaborate.
2. https://layer5.io/cloud-native-management/meshmap.
3. https://discuss.layer5.io/.
4. https://discuss.layer5.io/categories.
5. Update _http://slack.layer5.io/_ with  _https://slack.layer5.io/_ (https)

### Desired Situation
Provide a unique and concise title for each of the mentioned pages. Provide a relevant title.
- For information on how to create effective titles, please see [this Google article](https://support.google.com/webmasters/answer/35624).
- Remove the meta descriptions.
- You can also view the On-Page SEO Basics: [Meta Descriptions article](https://www.semrush.com/blog/on-page-seo-basics-meta-descriptions/)

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Shivam-AfA**
**Description**

This PR fixes #4041 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Shivam-AfA** [comment]
This PR also replaces http://slack.layer5.io/ with https://slack.layer5.io/ .

**Comment by l5io** [comment]
🚀 Preview for commit 5be0a6c118ee6d5d525a7a269325b1fae4fb284b at: https://645163a5126ea8007f82dcc0--layer5.netlify.app

File: src/pages/cloud-native-management/meshmap/collaborate.js [hunk 1]
```diff
@@ -13,5 +13,5 @@ const MeshmapCollaboratePage = () => {
 };
 export default MeshmapCollaboratePage;
 export const Head = () => {
-  return <SEO title="MeshMap" description="Collaboratively design and manage your Kubernetes clusters, service mesh deployments, and cloud native apps. Confidently configure your systems with MeshMap's visual topology and built-in design patterns." />;
+  return <SEO title="MeshMap Collaboration" description="Build an iterative design flow with live collaboration that keeps you in the loop whether you are working in the office or remotely." />;
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/pages/cloud-native-management/meshmap/collaborate.js:16]: Looks good to me.

File: src/pages/cloud-native-management/meshmap/design.js [hunk 2]
```diff
@@ -13,5 +13,5 @@ const MeshmapDesignPage = () => {
 };
 export default MeshmapDesignPage;
 export const Head = () => {
-  return <SEO title="MeshMap" description="Collaboratively design and manage your Kubernetes clusters, service mesh deployments, and cloud native apps. Confidently configure your systems with MeshMap's visual topology and built-in design patterns." />;
+  return <SEO title="MeshMap Designer" description="Design your infrastructure using a pallete of thousands of versioned Kubernetes components. Say goodbye to YAML configurations." />;
```
Comment (leecalcote) [inline src/pages/cloud-native-management/meshmap/design.js:16]:
```suggestion
  return <SEO title="MeshMap Designer" description="Design your infrastructure using a palette of thousands of versioned Kubernetes components. Say goodbye to YAML." />;
```

File: src/pages/cloud-native-management/meshmap/visualize.js [hunk 3]
```diff
@@ -17,5 +17,5 @@ export default MeshmapVisualizePage;
 
 
 export const Head = () => {
-  return <SEO title="MeshMap" description="Collaboratively design and manage your Kubernetes clusters, service mesh deployments, and cloud native apps. Confidently configure your systems with MeshMap's visual topology and built-in design patterns." />;
+  return <SEO title="MeshMap Visualizer" description="Import your existing Kubernetes, Helm, or Docker Compose applications, monitor various performance metrics and see your clusters in action." />;
```
Comment (leecalcote) [inline src/pages/cloud-native-management/meshmap/visualize.js:20]:
```suggestion
  return <SEO title="MeshMap Visualizer" description="Import your existing Kubernetes, Helm, or Docker Compose applications, monitor various performance metrics and see your clusters in-action." />;
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**